src/Game.cpp

```diff
@@ -456,6 +457,42 @@ bool Game::Run()
 		animationManager.Load(i, animations[i]);
 	}
 
+	// Load the campaign levels
+	const auto scriptsPath = _fileSystem->FindPath(_fileSystem->ScriptsPath());
+	levelManager.Load("campaign/Land1", "Land 1", scriptsPath / "Land1.txt", true);
+	levelManager.Load("campaign/Land2", "Land 2", scriptsPath / "Land2.txt", true);
+	levelManager.Load("campaign/Land3", "Land 3", scriptsPath / "Land3.txt", true);
+	levelManager.Load("campaign/Land4", "Land 4", scriptsPath / "Land4.txt", true);
+	levelManager.Load("campaign/Land5", "Land 5", scriptsPath / "Land5.txt", true);
```

Comment:
If campaign don't have names, the names here are kind of random. 
Perhaps the name can be set at load time allowing us to add and remove campaign levels dynamically.

Reply:
I could use the filename instead?

Reply:
The problem is that `Scripts` folder contains many .txt files that are not levels. if we iterate over the Scripts directory looking for .txt files and we add them to the menu, they'll be a lot of noise.

Reply:
All of the ones that aren't `InfoScript*.txt` are actually scene files though